2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,16 +60,249 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>
+                     <persName ref="PRS3082Chaine"/> mistakenly identified this a malkǝʾ-hymn as the one for
+                     <persName ref="PRS9848Walatta"/> in 
+                     <bibl>
+                        <ptr target="bm:Chaine1913RepPart1"/>
+                        <citedRange unit="page">202</citedRange>
+                     </bibl>, but correctly indentified this as a malkǝʾ-hymn to 
+                     <persName ref="PRS3698ehetaKr"/> in 
+                     <bibl>
+                        <ptr target="bm:Chaine1912Catalogue"/>
+                        <citedRange unit="page">57</citedRange>
+                        <citedRange unit="number">88</citedRange>
+                     </bibl>.
+                  </note>
+                  <bibl>
```

Comment:
this record must be moved one level up 

nesting must be logical 

bibliography list
-title 1
-title 2
-title 3

at the moment title 2 and title 3 are nested inside of title 1

the note concerning the identification should either be moved to `abstract ` (easier and clearer) or, if you want a commented bibliography, split for each bibliographic record
title 1 - wrong identification
title 2 - correct identification
title 3 - edition and further bibliography (you can also copy here the bibliography from Augustine's record)

Reply:
Moving the note to abstract is a good idea and makes bibliography list nested better. Thank you! I

Reply:
Please nest the bibliography correctly
listbibl opens and contains
 bibl opens closes
 bibl opens closes
 bibl opens closes
listbibl closes

it is not nested correctly yet

Reply:
Oh! Missed this too! thank you for pointing it out!

Reply:
thank you! 

what about repeating Chaine1912Catalogue? the list of bibliography must contain all quoted bibliography. it disappeared now.

what about bm:ContiRossini1912ZaraBuruk quoted by @smaugustine ?